src/components/LoginUser.js

```diff
@@ -0,0 +1,24 @@
+import { useState } from "react";
+
+const LoginUser = ( { user, setUser } ) => {
+    const [username, setUsername] = useState();
+    const [password, setPassword] = useState();
+
+    const submitHandler = (e) => {
+        e.preventDefault();
+        console.log("login user - ", username, password);
+        console.log("ToDo: LoginUser -> loginUser( {user}, setUser )")
```

Comment:
Now in here, you'll test whether 'user' is not undefined or null and that's how you know if he's logged in or not. 
- If he's logged in, it means the intention was to log out the user, so you call userLogout()
- If he's not logged in, then the intention was to log him in so you call userLogin( )